staker/stakerapp.go

```diff
@@ -506,6 +507,62 @@ func (app *App) mustSetTxSpentOnBtc(hash *chainhash.Hash) {
 	}
 }
 
+// SendPhase1Transaction receives the BTC staking transaction hash that
+// should be already in BTC and creates the necessary data to submit
+// the BTC delegation into the consumer chain
+func (app *App) SendPhase1Transaction(
+	stakerAddr btcutil.Address,
+	stkTxHash *chainhash.Hash,
+	tag []byte,
+	covenantPks []*secp256k1.PublicKey,
+	covenantQuorum uint32,
+) error {
+	stkTx, err := app.wc.Tx(stkTxHash)
+	if err != nil {
+		app.logger.WithError(err).Info("err get tx details")
+		return err
+	}
+
+	wireStkTx := stkTx.MsgTx()
+	parsedStakingTx, err := staking.ParseV0StakingTx(wireStkTx, tag, covenantPks, covenantQuorum, app.network)
+	if err != nil {
+		app.logger.WithError(err).Info("err parse staking Tx with global params")
+		return err
+	}
+
+	// unlock wallet to generate pop
+	// TODO: consider unlock/lock with defer
+	err = app.wc.UnlockWallet(defaultWalletUnlockTimeout)
+	if err != nil {
+		return err
+	}
+
+	pop, err := app.createPop(stakerAddr)
```

Comment:
ahh currently this type of pop works only for native segwit addresses, lets leave a comment about it.